### PR TITLE
Revert "Fix issue with blank strings and type extensions (#381)"

### PR DESCRIPTION
**Pull request by matthewmcgarvey**
This reverts commit 2788dff63119b14f64a010c1c2e67072685fa66e.

See https://github.com/luckyframework/avram/pull/381#issuecomment-738443183 as to why

### CODE REVIEW

File: src/avram/where.cr [hunk 1]
```diff
@@ -38,7 +38,7 @@ module Avram::Where
   end
 
   abstract class ValueHoldingSqlClause < SqlClause
-    getter value : String | Array(String) | Array(Int32) | Nil
```
Comment (jwoertink) [inline src/avram/where.cr:41]: All because of this... I'm still not sure why this messes with Lucky Params... 😝

**Review by jwoertink — APPROVED** [review]
Thanks for doing this. I think we still need it, but we may need to come up with a different implementation.

**Comment by jwoertink** [comment]
Hey @bcardiff if you get the chance, this should revert that change that broke your tests. Hopefully they all compile and run 😬

**Comment by bcardiff** [comment]
test-ecosystem is as happy as it can be regarding lucky 🎉  https://github.com/bcardiff/test-ecosystem/runs/1500476107?check_suite_focus=true . Thanks!

**Comment by jwoertink** [comment]
Sweet! That's good to know. We are looking in to how we can better catch stuff like this.

**Pull request merged**